README.md

```diff
@@ -16,6 +16,18 @@ To run project;
 npm run dev
 ```
 
+# Docker setup 
+
+To setup up with docker build the image using
+```
+sudo docker build . -t afri-frontend
+```
+
+To run containner 
+```
+sudo docker run --publish 3000:3000 afri-frontend
```

Comment:
can we have an npm script to run the container

Reply:
Thanks for the review @copy-ninja1